src/documents/tutorial-userInterfaceOptions/IntegratingUIOptionsStylingPreferences.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Integrating UI Options Styling Preferences
+category: Tutorials
+---
+
+Enactors modify a website to meet the preferences specified through UI Options. However, not all enactors modify
+the styling of a website. For those that do, stylesheets are provided to facilitate this application. Each stlying
+related enactor will provide its own stylesheet(s), so that an integrator can import only the styles for the enactors
+they are using in their particular configuration. Althernatively the `Enactors` or `Enactors_base` stylesheets can be
+used to quickly access all of the available styling enactors styles.
+
+## Enactor Stylesheets
+
+### Base Styleheets
+
+The styling related enactors use [CSS custom properties](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)
+as hooks for applying the styling. Some enactors such as `Text Size`, set their CSS custom properties via JavaScript,
+others supply them via classes in their related `*_base` stylesheet. For example the `Enhance Inputs` preference uses
```

Comment:
A little written-out example would help the reader a lot here - perhaps show what the CSS custom property looks like for Text Size, and show what the Enhance Inputs preference class looks like